selenium_tests/basic_test.py

```diff
@@ -38,6 +38,7 @@
 
 pytestmark = [
     pytest.mark.django_db,
+    pytest.mark.flaky(reruns=3),
```

Comment:
This should also go in `redirect_test.py`. Is there a way to have it apply to all tests in this directory?